test/parallel/test-repl-load-multiline.js

```diff
@@ -13,6 +13,7 @@ const expected = `${command}
 const getLunch = () =>
   placeOrder('tacos')
     .then(eat);
+
```

Comment:
Superfluous change?

Reply:
No, that's the bug fix mentioned here: https://github.com/nodejs/node/pull/28608#discussion_r302002539

Reply:
Ah, we're in a string template here, which wasn't obvious from the GitHub interface. I see now.